src/Steeltoe.All.sln

```diff
@@ -291,6 +291,10 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Steeltoe.Configuration.Clou
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Steeltoe.Configuration.CloudFoundry.ServiceBinding.Test", "Configuration\test\CloudFoundry.ServiceBinding.Test\Steeltoe.Configuration.CloudFoundry.ServiceBinding.Test.csproj", "{BF725F33-3361-4D9C-A1FA-27EEA79A09DA}"
 EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Steeltoe.Actuators", "..\..\Samples\Management\src\AllActuators\Actuators\Steeltoe.Actuators.csproj", "{D926E38F-6920-41B8-B76A-2841AB9E8D53}"
```

Comment:
Is there a branch where these samples can be tried? Are there any changes required in the samples besides changing package ref to project ref?

Reply:
I created this codespace: https://github.com/codespaces/friendly-train-v666495gqq3wp7g to see it working. 
If you go here you should be able to click on "beans " link to see output: https://friendly-train-v666495gqq3wp7g-5000.app.github.dev/

Reply:
I'm getting a 404 on both links.

Reply:
Don't worry about code space - best way is to probably use the actuator4.x branch in samples